### PR TITLE
Add favorite things

### PR DESCRIPTION
**Pull request by chansonnier**
Hi Andreea!

You should really look at this. I think this is an important addition. Please review and let me know what you think.

### CODE REVIEW

File: favorite_things.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+always gives me the benefit of the doubt
+does sweet things all the time
+enjoys seeing me enjoy something
+smells nice always
+is super hot
```
Comment (andreeahanson) [inline favorite_things.md]: Actually, I am really cold right now. Could you please change that? Thank you! @chansonnier

File: favorite_things.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+always gives me the benefit of the doubt
+does sweet things all the time
```
Comment (andreeahanson) [inline favorite_things.md]: actually, I MAKE sweet things all the time. That's why I'm not thin.

File: favorite_things.md (hunk 1 above)
Comment (chansonnier) [reply favorite_things.md]: Ok. I turned the thermostat up a few degrees. Please review and let me know what you think.

File: favorite_things.md (hunk 2 above)
Comment (chansonnier) [reply favorite_things.md]: You ARE thin!

File: favorite_things.md (hunk 1 above)
Comment (andreeahanson) [reply favorite_things.md]: No, you didn't. So please adjust this line of code, as it is unacceptable. Thank you!

File: favorite_things.md (hunk 2 above)
Comment (andreeahanson) [reply favorite_things.md]: No, I'm not. But let's pretend I am.

File: favorite_things.md [hunk 3]
```diff
@@ -2,6 +2,6 @@ always gives me the benefit of the doubt
 does sweet things all the time
 enjoys seeing me enjoy something
 smells nice always
-is super hot
+is cold at the moment but the thermostat was turned up and should be warmer soon. Also is super hot.
```
Comment (andreeahanson) [inline favorite_things.md]: 😄

**Pull request merged**